data.sql

```diff
@@ -62,4 +62,43 @@ UPDATE animals SET owners_id = 3 WHERE name IN('Devimon', 'Plantmon');
 UPDATE animals SET owners_id = 4 WHERE name IN('Charmander', 'Squirtle','Blossom');
 UPDATE animals SET owners_id = 5 WHERE name IN('Angemon', 'Boarmon');
 COMMIT;
-SELECT * FROM animals;
\ No newline at end of file
+SELECT * FROM animals;
+
+-- insert into vets
+INSERT INTO vets(name, age, graduation_date)
+VALUES('William Tatcher', 45, '2000-03-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08')
+
+-- insert into specializations
+INSERT INTO specializations(species_id, vet_id)
+VALUES((SELECT id FROM species WHERE name = 'Pokemon'),(SELECT id FROM vets WHERE name = 'William Tatcher')),
+((SELECT id FROM species WHERE name = 'Pokemon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+((SELECT id FROM species WHERE name = 'Digimon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez')),
+((SELECT id FROM species WHERE name = 'Digimon'),(SELECT id FROM vets WHERE name = 'Jack Harkness'))
+
+-- insert into visits
+INSERT INTO visits(animal_id, vet_id, date)
+VALUES((SELECT animalid FROM animals WHERE name = 'Augmon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+((SELECT animalid FROM animals WHERE name = 'Augmon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
```

Comment:
- [ ] Kindly fix the name of the pokemon to match with the provided which is `Agumon`